content/docs/reference-glossary.md

```diff
@@ -47,27 +47,26 @@ ReactDOM.render(
 );
 ```  
 
-## [Elements](/docs/rendering-elements.html)
+## [Elementos](/docs/rendering-elements.html)
 
-React elements are the building blocks of React applications. One might confuse elements with a more widely known concept of "components". An element describes what you want to see on the screen. React elements are immutable.
+Los elementos de React son los bloques de construcción de una aplicación de React. Uno podría confundir los elementos con el concepto más ampliamente conocido de "componentes". Un elemento describe lo que quieres ver en pantalla. Los elementos de React son immutables.
```

Comment:
Should be `inmutables` instead of `immutables`

Reply:
Thanks!